lib/spring/test/acceptance_test.rb

```diff
@@ -94,6 +102,22 @@ def without_gem(name)
         refute app.spring_env.server_running?
       end
 
+      test "running spring status for rails/rake commands" do
```

Comment:
these descriptions could make it easier for other users to understand what is being tested:

```
test "tells the user that spring is being used when used automatically via binstubs"
test "does not tell the user that spring is being used when the user used spring"
test "does not tell the user that spring is being when used automatically via binstubs but quiet is set to true"
```